### DISCUSSION TITLE
Possible to run NocoDB as part of a website?

### DISCUSSION THREAD

**Discussion by mainrs** [nocodb/nocodb#286] [Q&A] [answered]
Is it possible to embed NocoDB into a website? I am thinking about how to present the data for a tool of mine and I personally think that NocoDB would be a great option!

Browsers do support IndexDB nowadays. Would that be feasible as a backend for NocoDB? Running it inside the browser would mean that not all of the components of NocoDB would be needed. For example authentication and managing users/access is probably unneccessary in most cases, as the table would operate locally and can't be shared anyway.

**Comment by o1lab** [comment, marked as answer]
You can embed share view. This allows embedding only that particular table.

**Reply by mainrs** [reply]
That would still mean that I'd have to have a server running a database somewhere. Support for local databases from within a browser (like IndexDB) is not implemented, right?

Would it, in theory, be possible? Or is the code written in such a way that it is actually hard to do that?

**Reply by o1lab** [reply]
I see your point. Currently, we ve no plans for indexdb.

**Comment by o1lab** [comment]
https://youtu.be/TVQgAFieIow

**Comment by holms** [comment]
@o1lab thank you for the great product :) is it possible to have the same view, but online? I have no problem hosting database on my own server or using some digital ocean mysql as a service. I just would like to give a link with a car parts for some people online, it's very handy. Huge potential to use such page for ebay for example.

**Reply by mertmit** [reply]
Our shared view is exactly what you are looking for, you can share any view from NocoDB for others to view.
You can check details in our docs (https://docs.nocodb.com/views/share-view)
